src/Sulu/Component/Content/Compat/RoutableStructureInterface.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Content\Compat;
+
+interface RoutableStructureInterface
```

Comment:
@danrot not sure about this name 🤔

Reply:
Not sure about the name either, and I don't really understand why we need that... Is there a pressing reason for this? It's a change with a certain risk, and to me it feels like this part will change a lot in the future anyway :see_no_evil: 

If there is a pressing reason, why not moving that part to the `StructureInterface`? Don't structures always have these properties as well? Or is there something like a `SnippetStructure` as well?